packages/components/link-box/src/LinkBox.doc.mdx

```diff
@@ -37,6 +37,8 @@ Simply wrapping an entire component in an `a` tag may seem convenient for linkin
 
 ### Nesting
 
-Any nested links and buttons inside `LinkBox` are elevated to the top using `z-index` to ensure proper navigation between links.
+If your linkbox contains interactive elements inside of it, wrap them with `LinkBox.Raised`.
 
-<Canvas of={stories.Nesting} />
\ No newline at end of file
+It will raise their `z-index` to stay above the main link of your `LinkBox`.
```

Comment:
Maybe we should clarify that a single `LinkBox` can contain only one `LinkBox.Link`